notebooks/E2Ev2.ipynb

```diff
@@ -0,0 +1,350 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
```

Comment:
Architecturally, this notebook performs a QA stage for all datasets before continuing to the next stage (i.e., first download all data, then do all the extraction, ...). This is nice for an initial presentation. Nevertheless, for the QA process it is more important to have a function that performs all QA stages for one dataset and call that function multiple times.